src/AppDevRouter.js

```diff
@@ -10,104 +11,104 @@ export type RequestType = 'GET' | 'POST' | 'DELETE';
 
 /**
  * AppDevResponse - the response from an HTTP request
- * 
+ *
  * Wraps a `success` field around the response data
  */
 class AppDevResponse<T> {
   success: boolean;
   data: T;
 
-  constructor (success: boolean, data: T) {
-    this.success = success;
-    this.data = data;
+  constructor(success: boolean, data: T) {
+      this.success = success;
+      this.data = data;
   }
 }
 
 /**
  * AppDevRouter - cleanly create an Express Router object using inheritance
  *
- * Subclasses can simply specify the HTTP method, the path, and a response 
- * hook to compute response data. This pattern is cleaner than raw Express 
+ * Subclasses can simply specify the HTTP method, the path, and a response
+ * hook to compute response data. This pattern is cleaner than raw Express
  * Router initialization with callbacks.
  */
 class AppDevRouter<T> {
   router: Router;
-  requestType: RequestType;
+  requestTypes: Array<RequestType>;
 
   /**
    * Subclasses must call this constructor and pass in the HTTP method
    */
-  constructor(type: RequestType) {
-    this.router = new Router();
-    this.requestType = type;
+  constructor(types: RequestType) {
+      this.router = new Router();
```

Comment:
I know that in Java it's bad to use "this"--is using "this" common style in Node? just curious

Reply:
is it bad in Java b/c its often not necessary? i'm not sure if its common in javscript but i don't think its something enforced

Reply:
it's pretty common in js